src/main.c

```diff
@@ -1,3 +1,5 @@
+#include "wren_common.h"
```

Comment:
To be honest, the `main.c` change wasn't needed. That is, since it resides in the wren-library client side, it should be the client duty to ensure not to get warnings.

I'll revert the change for `main.c`.